src/app/business/home/home.component.ts

```diff
@@ -85,7 +86,7 @@ export class HomeComponent implements OnInit {
         this.backendForm = this.fb.group({
             "name":['', {validators:[Validators.required,Utils.isExisted(this.allBackendNameForCheck)]}],
             "type":['',{validators:[Validators.required]}],
-            "region":['',{validators:[Validators.required], updateOn:'change'}],
+            "region":['',{updateOn:'change'}],
```

Comment:
is region required for other storage backend options?
if yes, will this statement not remove the required validator for the other types?

Reply:
The other storage backend types require the `Region` field.
Since the Region is not needed only for CEPH type, you should disable the field and set required to false only for this type. For the other types it must still be a required field.